index.html

```diff
@@ -6,54 +6,68 @@
     <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <header class="index">
+    <header class="index-header">
       <div class="container">
-        <img  class="logo-index" src="http://placehold.it/150x100" alt="">
-        <nav>
-          <ul class="index-menu">
-            <li><a href="#">СТУДИЯ</a></li>
-            <li><a href="#">КЛИЕНТЫ</a></li>
-            <li><a href="#">МАГАЗИН</a></li>
-            <li><a href="#">КОНТАКТЫ</a></li>
-          </ul>
-        </nav>
-        <div class="user-block">
-          <a class="login" href="#">ВХОД</a>
+        <div class="upper clearfix">
+          <div class="logo">
+            <img  class="logo-index" src="img/nerds-logo.png" alt="">
+          </div>
+          <nav class="menu">
+            <ul class="index-menu">
+              <li><a href="#">СТУДИЯ</a></li>
+              <li><a href="#">КЛИЕНТЫ</a></li>
+              <li><a href="#">МАГАЗИН</a></li>
+              <li><a href="#">КОНТАКТЫ</a></li>
+            </ul>
+          </nav>
+          <div class="cart-block">
+            <a class="login" href="#">КОРЗИНА</a>
+          </div>
         </div>
         <div class="slider">
-          <div class="slide-first">
-            <span class="moto">Долго, дорого, скрупулезно.</span>
-            <p>Математически выверенный дизайн для вашего сайта или мобильного приложения.</p>
-            <a href="#" class="slider-btn">УЗНАТЬ БОЛЬШЕ</a>
+          <div class="slide slide-first">
+            <div class="slider-left">
+              <span class="moto">Долго, дорого, скрупулезно.</span>
+              <p>Математически выверенный дизайн для вашего сайта или мобильного приложения.</p>
+              <a href="#" class="btn slider-btn">УЗНАТЬ БОЛЬШЕ</a>
+            </div>
+          </div>
+          <div class="slide slide-second">
+
+          </div>
+          <div class="slide slide-third">
+
+          </div>
+          <div class="controls">
             <input type="radio" name="slider" checked>
             <input type="radio" name="slider">
             <input type="radio" name="slider">
           </div>
         </div>
       </div>
     </header>
-    <main>
-      <section class="features">
+    <main class="container">
+      <section class="features clearfix">
         <div class="feature web">
-          <img src="http://placehold.it/150x100" alt="">
+          <img src="img/feature-image-1.jpg" alt="">
           <span class="features">ВЕБ-САЙТЫ</span>
```

Comment:
This header shouldn't have the same class as the "features" section. It should be named something like "feature-header" or "feature-name".